client/src/ui/molecules/main-menu/index.tsx

```diff
@@ -73,128 +77,128 @@ export default function MainMenu() {
     });
   });
 
-  // The menus array includes objects that define the set of
-  // menus displayed by this header component. The data structure
-  // happen after the string catalog is available. And we're using
-  // useMemo() so that localization only happens when the locale
-  // changes.
-  const menus = useMemo(
-    () => [
-      {
-        label: "Technologies",
-        labelId: "technologies",
-        items: [
-          {
-            url: `/${locale}/docs/Web`,
-            label: "Technologies Overview",
+  const menus = [
+    {
+      label: "Technologies",
+      labelId: "technologies",
+      items: [
+        {
+          url: `/${locale}/docs/Web`,
+          label: "Technologies Overview",
+        },
+        {
+          url: `/${locale}/docs/Web/HTML`,
+          label: "HTML",
+        },
+        {
+          url: `/${locale}/docs/Web/CSS`,
+          label: "CSS",
+        },
+        {
+          url: `/${locale}/docs/Web/JavaScript`,
+          label: "JavaScript",
+        },
+        {
+          url: `/${locale}/docs/Web/Guide/Graphics`,
+          label: "Graphics",
+        },
+        {
+          url: `/${locale}/docs/Web/HTTP`,
+          label: "HTTP",
+        },
+        {
+          url: `/${locale}/docs/Web/API`,
+          label: "APIs",
+        },
+        {
+          url: `/${locale}/docs/Mozilla/Add-ons/WebExtensions`,
+          label: "Browser Extensions",
+        },
+        {
+          url: `/${locale}/docs/Web/MathML`,
+          label: "MathML",
+        },
+      ],
+    },
+    {
+      label: "References & Guides",
+      labelId: "references-guides",
+      items: [
+        {
+          url: `/${locale}/docs/Learn`,
+          label: "Learn web development",
+        },
+        {
+          url: `/${locale}/docs/Web/Tutorials`,
+          label: "Tutorials",
+        },
+        {
+          url: `/${locale}/docs/Web/Reference`,
+          label: "References",
+        },
+        {
+          url: `/${locale}/docs/Web/Guide`,
+          label: "Developer Guides",
+        },
+        {
+          url: `/${locale}/docs/Web/Accessibility`,
+          label: "Accessibility",
+        },
+        {
+          url: `/${locale}/docs/Games`,
+          label: "Game development",
+        },
+        {
+          url: `/${locale}/docs/Web`,
+          label: "...more docs",
+        },
+      ],
+    },
+    {
+      label: "Feedback",
+      labelId: "feedback",
+      items: [
+        {
+          url: `/${locale}/docs/MDN/Feedback`,
```

Comment:
This redirects to `/${locale}/docs/MDN/Contribute/Feedback` so I think we should go directly there:
```suggestion
          url: `/${locale}/docs/MDN/Contribute/Feedback`,
```